bop_toolkit_lib/dataset_params.py

```diff
@@ -384,49 +407,90 @@ def get_split_params(datasets_path, dataset_name, split, split_type=None):
+                ),
+                # Path template to a depth image.
+                "depth_tpath": join(
+                    split_path, "{scene_id:06d}", "depth", "{im_id:06d}" + depth_ext
+                ),
+                # Path template to a file with per-image camera parameters.
+                "scene_camera_tpath": join(
+                    split_path, "{scene_id:06d}", "scene_camera.json"
+                ),
+                # Path template to a file with GT annotations.
+                "scene_gt_tpath": join(
+                    split_path, "{scene_id:06d}", "scene_gt.json"
+                ),
+                # Path template to a file with meta information about the GT annotations.
+                "scene_gt_info_tpath": join(
+                    split_path, "{scene_id:06d}", "scene_gt_info.json"
+                ),
+                # Path template to a file with the coco GT annotations.
+                "scene_gt_coco_tpath": join(
+                    split_path, "{scene_id:06d}", "scene_gt_coco.json"
+                ),
+                # Path template to a mask of the full object silhouette.
+                "mask_tpath": join(
+                    split_path, "{scene_id:06d}", "mask", "{im_id:06d}_{gt_id:06d}.png"
+                ),
+                # Path template to a mask of the visible part of an object silhouette.
+                "mask_visib_tpath": join(
+                    split_path,
+                    "{scene_id:06d}",
+                    "mask_visib",
+                    "{im_id:06d}_{gt_id:06d}.png",
+                ),
+            }
+        )
+
+    else:
+        for moda in p["im_modalities"]:
+            p.update(
+                {
+                    # Path template to modality image.
+                    "{}_tpath".format(moda): join(
+                        split_path, "{scene_id:06d}", moda, "{im_id:06d}" + rgb_ext
+                    ),
+                    # Path template to a file with per-image camera parameters.
+                    "scene_camera_{}_tpath".format(moda): join(
+                        split_path, "{scene_id:06d}", "scene_camera_{}.json".format(moda)
+                    ),
+                    # Path template to a file with GT annotations.
+                    "scene_gt_{}_tpath".format(moda): join(
+                        split_path, "{scene_id:06d}", "scene_gt_{}.json".format(moda)
+                    ),
+                    # Path template to a file with meta information about the GT annotations.
+                    "scene_gt_info_{}_tpath".format(moda): join(
+                        split_path, "{scene_id:06d}", "scene_gt_info_{}.json".format(moda)
+                    ),
+                    # # Path template to a file with the coco GT annotations.  TODO: should be included?
+                    # "scene_gt_coco_tpath": join(
+                    #     split_path, "{scene_id:06d}", "scene_gt_coco.json"
+                    # ),
```

Comment:
This is required if we want to evaluate 2D detection / segmentation.

Reply:
simply run :
```
python scripts/calc_gt_coco.py
```

Reply:
Ok, `calc_gt_coco.py` loads the full mask and visible of each ground-truth, computes amodal/modal bounding boxes and stores results in coco format. So, I just need to adapt slightly `calc_gt_coco.py` to the new HOT3D file organization ("scene_gt_coco.json" -> "scene_gt_coco_rgb.json") and it should be fine.

Reply:
Adapted `calc_gt_coco.py` and `eval_bop22_coco.py`, not checked against ground truth estimates though for hot3d.